public/html/register.html

```diff
@@ -14,7 +14,8 @@
 <body>
     <div class="container">
         <div class="content-main">
-        <!-- Step 1: Email -->
+            <form id="registration-form">
```

Comment:
Added for id to be able to pull the data out before making a fetch call to the b/e.